script/build.js

```diff
@@ -128,6 +128,8 @@ if (options.buildtype === 'production') {
   });
   ignoreList.push('veteran-id-card/how-to-get.md');
   ignoreList.push('veteran-id-card/how-to-upload-photo.md');
+  ignoreList.push('education/opt-out.md');
```

Comment:
Per this [comment](https://github.com/department-of-veterans-affairs/vets-website/pull/7975#pullrequestreview-135554685) the url is changing to `vets.gov/education/opt-out-information-sharing`

Reply:
Thanks @rianfowler, updated.